apps/launch/components/MyStakingContracts/Create/index.spec.tsx

```diff
@@ -120,6 +120,12 @@ describe('<CreateStakingContract />', () => {
     await userEvent.type(rewardsPerSecondInput, '100');
     expect(rewardsPerSecondInput).toHaveValue('100');
 
+    const minStakingDepositInput = screen.getByLabelText('Minimum service staking deposit, OLAS');
+    // clear first, because it has default value
+    await userEvent.clear(minStakingDepositInput);
+    await userEvent.type(minStakingDepositInput, '1000');
+    expect(minStakingDepositInput).toHaveValue('1000');
```

Comment:
Why is this needed at all? Can it be just read from the contract every time?

Reply:
it's just a test......